code/datums/martial/plasma_fist.dm

```diff
@@ -93,7 +93,7 @@
 		var/mob/living/carbon/human/human_attacker = user
 		human_attacker.set_species(/datum/species/plasmaman)
 		ADD_TRAIT(human_attacker, TRAIT_FORCED_STANDING, type)
-		human_attacker.dna.species.species_traits += TRAIT_BOMBIMMUNE
```

Comment:
Lol this wouldn’t have even worked, since this adds it to the species datum and not the mob and the species datum only updates the mob on gain and on lose